CMakeLists.txt

```diff
@@ -180,14 +180,17 @@ endif()
 
 find_package(SDL3 REQUIRED CONFIG REQUIRED COMPONENTS SDL3)
 find_package(glm REQUIRED)
-if (TARGET httlib::httplib)
+find_package(httplib QUIET)
+if (TARGET httplib::httplib)
   add_library(httplib ALIAS httplib::httplib)
 else()
   # Some Linux distributions do not package httplibConfig.cmake, so find the header manually
   find_file(CPP_HTTPLIB_HEADER httplib.h REQUIRED)
+  find_library(CPP_HTTPLIB_LIB cpp-httplib REQUIRED)
```

Comment:
Some distributions, for instance fedora, won't ship the shared library file (see https://packages.fedoraproject.org/pkgs/cpp-httplib/cpp-httplib-devel/fedora-42.html#files), because it is supposed to be header-only. This library packaging across distribution is a mess...